docs/storage/README.md

```diff
@@ -0,0 +1,14 @@
+# cAdvisor Storage Plugins
+
+cAdvisor supports exporting stats to various storage driver plugins. To enable a storage driver, set the `-storage_driver` flag.
```

Comment:
Doesn't matter (I don't like go's flag library), but 1-dash seems more consistent with the other storage docs.